crates/re_renderer/src/renderer/point_cloud.rs

```diff
@@ -397,7 +399,7 @@ impl PointCloudDrawData {
                 batches
                     .iter()
                     .map(|batch_info| gpu_data::BatchUniformBuffer {
-                        world_from_obj: batch_info.world_from_obj.into(),
+                        world_from_obj: glam::Mat4::from(batch_info.world_from_obj).into(),
```

Comment:
Let's add a `impl From<Affine3A> for wgpu_buffer_types::Mat4` helper instead so this could just be an `into`